dist/index.js

```diff
@@ -1,19 +1,19 @@
e.port?n=Number(e.port):e.protocol==="http:"?n=80:e.protocol==="https:"&&(n=443);let t=[e.hostname.toUpperCase()];typeof n=="number"&&t.push(`${t[0]}:${n}`);for(let o of r.split(",").map(i=>i.trim().toUpperCase()).filter(i=>i))if(t.some(i=>i===o))return!0;return!1}nn.checkBypass=ci});var mi=A(Ve=>{"use strict";var e_=require("net"),Xf=require("tls"),zt=require("http"),di=require("https"),Qf=require("events"),r_=require("assert"),Zf=require("util");Ve.httpOverHttp=em;Ve.httpsOverHttp=rm;Ve.httpOverHttps=nm;Ve.httpsOverHttps=tm;function em(e){var r=new ye(e);return r.request=zt.request,r}function rm(e){var r=new ye(e);return r.request=zt.request,r.createSocket=li,r.defaultPort=443,r}function nm(e){var r=new ye(e);return r.request=di.request,r}function tm(e){var r=new ye(e);return r.request=di.request,r.createSocket=li,r.defaultPort=443,r}function ye(e){var r=this;r.options=e||{},r.proxyOptions=r.options.proxy||{},r.maxSockets=r.options.maxSockets||zt.Agent.defaultMaxSockets,r.requests=[],r.sockets=[],r.on("free",function(t,o,i,a){for(var s=fi(o,i,a),c=0,l=r.requests.length;c<l;++c){var d=r.requests[c];if(d.host===s.host&&d.port===s.port){r.requests.splice(c,1),d.request.onSocket(t);return}}t.destroy(),r.removeSocket(t)})}Zf.inherits(ye,Qf.EventEmitter);ye.prototype.addRequest=function(r,n,t,o){var i=this,a=jt({request:r},i.options,fi(n,t,o));if(i.sockets.length>=this.maxSockets){i.requests.push(a);return}i.createSocket(a,function(s){s.on("free",c),s.on("close",l),s.on("agentRemove",l),r.onSocket(s);function c(){i.emit("free",s,a)}function l(d){i.removeSocket(s),s.removeListener("free",c),s.removeListener("close",l),s.removeListener("agentRemove",l)}})};ye.prototype.createSocket=function(r,n){var t=this,o={};t.sockets.push(o);var i=jt({},t.proxyOptions,{method:"CONNECT",path:r.host+":"+r.port,agent:!1,headers:{host:r.host+":"+r.port}});r.localAddress&&(i.localAddress=r.localAddress),i.proxyAuth&&(i.headers=i.headers||{},i.headers["Proxy-Authorization"]="Basic "+new Buffer(i.proxyAuth).toString("base64")),Re("making CONNECT request");var a=t.request(i);a.useChunkedEncodingByDefault=!1,a.once("response",s),a.once("upgrade",c),a.once("connect",l),a.once("error",d),a.end();function s(f){f.upgrade=!0}function c(f,m,p){process.nextTick(function(){l(f,m,p)})}function l(f,m,p){if(a.removeAllListeners(),m.removeAllListeners(),f.statusCode!==200){Re("tunneling socket could not be established, statusCode=%d",f.statusCode),m.destroy();var v=new Error("tunneling socket could not be established, statusCode="+f.statusCode);v.code="ECONNRESET",r.request.emit("error",v),t.removeSocket(o);return}if(p.length>0){Re("got illegal response body from proxy"),m.destroy();var v=new Error("got illegal response body from proxy");v.code="ECONNRESET",r.request.emit("error",v),t.removeSocket(o);return}return Re("tunneling connection has established"),t.sockets[t.sockets.indexOf(o)]=m,n(m)}function d(f){a.removeAllListeners(),Re(`tunneling socket could not be established, cause=%s
```

Comment:
Should we be including the dist folder?

Reply:
yes - this module pattern depends on dist/index.js being the entry point for the GitHub action